src/app/components/Header/Header.tsx

```diff
@@ -6,8 +6,33 @@ import classNames from 'classnames';
 import { basePath } from '../../utils/constants';
 import { NavigationLink } from '../NavigationLink/NavigationLink';
 import styles from './Header.module.scss';
+import Hero from '../Hero/Hero';
+import { usePathname } from 'next/navigation';
 
 export default function Header() {
+  interface HeroContent {
+    heroClass?: string;
+    heroHeader: string;
+    heroSubheader: string;
+  }
+
+  type SpecialContent = {
+    [key: string]: HeroContent;
+  };
+
+  const specialContent: SpecialContent = {
+    '/': {
+      heroClass: 'homepage-hero',
+      heroHeader: `Improve the way your jurisdiction collects, processes, and
+      views public health data`,
+      heroSubheader: `Turn your jurisdiction's data into meaningful intelligence that drives
+      timely public health action using CDC's free, cloud-based products built
+      from Data Integration Building Blocks, or DIBBs.`,
+    },
+  };
```

Comment:
Can we move these definitions outside of `Header` (maybe just above)?

Reply:
yep